crates/router/src/connector/stax/transformers.rs

```diff
@@ -168,10 +300,20 @@ impl TryFrom<types::RefundsResponseRouterData<api::Execute, RefundResponse>>
     fn try_from(
         item: types::RefundsResponseRouterData<api::Execute, RefundResponse>,
     ) -> Result<Self, Self::Error> {
+        let refund_txn = item
```

Comment:
Let's rewrite this logic as in pick the right refund object using amount and created timestamp

Reply:
Done